docs/spec.md

```diff
@@ -4184,7 +4184,7 @@ where:
 * `padded_inputs = pad(inputs..., init_values..., padding[:, 0], padding[:, 1],
   base_dilations - 1)`.
 * `window_start = result_index * window_strides`.
-* `window_end = window_start + window_dimensions + window_dilations - 1`.
+* `window_end = window_start + window_dimensions + (window_dimensions - 1) * (window_dilations - 1)`.
```

Comment:
Also, let's add tests - both for the bug that this PR fixes and for the zero case.

Reply:
Added a case for the bug, but not the zero case as noted in my comment above.